gapic-generator-cloud/templates/cloud/service/client/_self_configure.erb

```diff
@@ -1,6 +1,14 @@
 <%- assert_locals service -%>
 @configure ||= begin
-  parent_config = <%= service.gem.namespace %>.configure
+  namespace = <%= service.namespace %>.name.split "::"
```

Comment:
Consider doing the splitting in the codegen itself, so the generated code just includes an array literal. e.g.
```
namespace = <%= service.namespace.split("::").inspect %>
```

which should generate, e.g.
```ruby
namespace = ["Google", "Cloud", "Language", "V1"]
```